### ISSUE TITLE
Localization packaging issue in lutris 0.5.6 Debian/Ubuntu package

### ISSUE THREAD

**Issue by DaDummy** [lutris/lutris#2749]
**Describe the bug**

There seems to be a localization issue with the lutris 0.5.6 Debian/Ubuntu package version number.

I currently have the 0.5.6 beta package from Discord installed which appears to carry the exact same version number, but is displayed correctly as can be seen.

Maybe this is related to both carrying the same version number? (Can I/somebody reproduce this with 0.5.5 installed?)

**Expected behavior**

The version number is shown correctly.

**Current behavior**

`%2(I18N_ARGUMENT_MISSING)` is shown instead.

**Steps to reproduce**

- Try to install the latest Update using KDE Plasma Discover in Kubuntu 19.10 - might affect other GUI package managers aswell. aptitude is not affected and displays the correct version number.

**Lutris debugging output**

Does not apply

**System information**

```json
{
  "comment": "",
  "system": {
    "gpus": [
      {
        "DRIVER": "amdgpu",
        "PCI_ID": "1002:67DF",
        "PCI_SUBSYS_ID": "1DA2:E366",
        "PCI_CLASS": "30000",
        "PCI_SLOT_NAME": "0000:01:00.0",
        "MODALIAS": "pci:v00001002d000067DFsv00001DA2sd0000E366bc03sc00i00"
      }
    ],
    "env": {
      "KDE_FULL_SESSION": "true",
      "GS_LIB": "/home/chris/.fonts",
      "USER": "chris",
      "PAM_KWALLET5_LOGIN": "<redacted>",
      "LC_TIME": "de_DE.UTF-8",
      "LANGUAGE": "en_US:de",
      "XDG_SEAT": "seat0",
      "QT_SCREEN_SCALE_FACTORS": "DisplayPort-0=1;DisplayPort-1=1;HDMI-A-0=1;HDMI-A-1=1;DVI-D-0=1;",
      "XDG_SESSION_TYPE": "x11",
      "SSH_AGENT_PID": "2422",
      "SHLVL": "0",
      "HOME": "/home/chris",
      "DESKTOP_SESSION": "plasma",
      "XDG_SEAT_PATH": "/org/freedesktop/DisplayManager/Seat0",
      "KDE_SESSION_VERSION": "5",
      "LC_MONETARY": "de_DE.UTF-8",
      "DBUS_SESSION_BUS_ADDRESS": "unix:path=/run/user/1000/bus",
      "LIBVIRT_DEFAULT_URI": "qemu:///system",
      "MANDATORY_PATH": "/usr/share/gconf/plasma.mandatory.path",
      "LOGNAME": "chris",
      "QT_AUTO_SCREEN_SCALE_FACTOR": "0",
      "XDG_SESSION_CLASS": "user",
      "DEFAULTS_PATH": "/usr/share/gconf/plasma.default.path",
      "XDG_SESSION_ID": "4",
      "PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/chris/.dotnet/tools:/opt/rocm-3.3.0/bin/",
      "XDG_SESSION_PATH": "/org/freedesktop/DisplayManager/Session1",
      "LC_ADDRESS": "de_DE.UTF-8",
      "XDG_RUNTIME_DIR": "/run/user/1000",
      "XCURSOR_THEME": "breeze_cursors",
      "DISPLAY": ":0",
      "LANG": "de_DE.UTF-8",
      "XDG_CURRENT_DESKTOP": "KDE",
      "LC_TELEPHONE": "de_DE.UTF-8",
      "XDG_SESSION_DESKTOP": "KDE",
      "XAUTHORITY": "<redacted>",
      "SSH_AUTH_SOCK": "<redacted>",
      "SHELL": "/bin/bash",
      "LC_NAME": "de_DE.UTF-8",
      "LC_MEASUREMENT": "de_DE.UTF-8",
      "GPG_AGENT_INFO": "<redacted>",
      "LC_IDENTIFICATION": "de_DE.UTF-8",
      "XDG_VTNR": "1",
      "PWD": "/home/chris",
      "XDG_CONFIG_DIRS": "/etc/xdg/xdg-plasma:/etc/xdg:/usr/share/kubuntu-default-settings/kf5-settings",
      "XDG_DATA_DIRS": "/usr/share/plasma:/usr/local/share:/usr/share:/var/lib/snapd/desktop:/var/lib/snapd/desktop",
      "LC_NUMERIC": "de_DE.UTF-8",
      "LC_PAPER": "de_DE.UTF-8",
      "KDE_SESSION_UID": "1000",
      "SESSION_MANAGER": "<redacted>",
      "DESKTOP_STARTUP_ID": "<redacted>"
    },
    "missing_libs": {
      "OPENGL": [
        [],
        []
      ],
      "VULKAN": [
        [],
        []
      ],
      "WINE": [
        [],
        []
      ],
      "GAMEMODE": [
        [
          "libgamemodeauto.so"
        ],
        [
          "libgamemodeauto.so"
        ]
      ],
      "RADEON": [
        [],
        []
      ]
    },
    "cpus": [
      {
        "processor": "0",
        "vendor_id": "GenuineIntel",
        "cpu family": "6",
        "model": "60",
        "model name": "Intel(R) Xeon(R) CPU E3-1240 v3 @ 3.40GHz",
        "stepping": "3",
        "microcode": "0x27",
        "cpu MHz": "2708.972",
        "cache size": "8192 KB",
        "physical id": "0",
        "siblings": "8",
        "core id": "0",
        "cpu cores": "4",
        "apicid": "0",
        "initial apicid": "0",
        "fpu": "yes",
        "fpu_exception": "yes",
        "cpuid level": "13",
        "wp": "yes",
        "flags": "fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt dtherm ida arat pln pts md_clear flush_l1d",
        "bugs": "cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit",
        "bogomips": "6800.16",
        "clflush size": "64",
        "cache_alignment": "64",
        "address sizes": "39 bits physical, 48 bits virtual",
        "power management": ""
      },
      {
        "processor": "1",
        "vendor_id": "GenuineIntel",
        "cpu family": "6",
        "model": "60",
        "model name": "Intel(R) Xeon(R) CPU E3-1240 v3 @ 3.40GHz",
        "stepping": "3",
        "microcode": "0x27",
        "cpu MHz": "2830.560",
        "cache size": "8192 KB",
        "physical id": "0",
        "siblings": "8",
        "core id": "1",
        "cpu cores": "4",
        "apicid": "2",
        "initial apicid": "2",
        "fpu": "yes",
        "fpu_exception": "yes",
        "cpuid level": "13",
        "wp": "yes",
        "flags": "fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt dtherm ida arat pln pts md_clear flush_l1d",
        "bugs": "cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit",
        "bogomips": "6800.16",
        "clflush size": "64",
        "cache_alignment": "64",
        "address sizes": "39 bits physical, 48 bits virtual",
        "power management": ""
      },
      {
        "processor": "2",
        "vendor_id": "GenuineIntel",
        "cpu family": "6",
        "model": "60",
        "model name": "Intel(R) Xeon(R) CPU E3-1240 v3 @ 3.40GHz",
        "stepping": "3",
        "microcode": "0x27",
        "cpu MHz": "2661.192",
        "cache size": "8192 KB",
        "physical id": "0",
        "siblings": "8",
        "core id": "2",
        "cpu cores": "4",
        "apicid": "4",
        "initial apicid": "4",
        "fpu": "yes",
        "fpu_exception": "yes",
        "cpuid level": "13",
        "wp": "yes",
        "flags": "fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt dtherm ida arat pln pts md_clear flush_l1d",
        "bugs": "cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit",
        "bogomips": "6800.16",
        "clflush size": "64",
        "cache_alignment": "64",
        "address sizes": "39 bits physical, 48 bits virtual",
        "power management": ""
      },
      {
        "processor": "3",
        "vendor_id": "GenuineIntel",
        "cpu family": "6",
        "model": "60",
        "model name": "Intel(R) Xeon(R) CPU E3-1240 v3 @ 3.40GHz",
        "stepping": "3",
        "microcode": "0x27",
        "cpu MHz": "2813.932",
        "cache size": "8192 KB",
        "physical id": "0",
        "siblings": "8",
        "core id": "3",
        "cpu cores": "4",
        "apicid": "6",
        "initial apicid": "6",
        "fpu": "yes",
        "fpu_exception": "yes",
        "cpuid level": "13",
        "wp": "yes",
        "flags": "fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt dtherm ida arat pln pts md_clear flush_l1d",
        "bugs": "cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit",
        "bogomips": "6800.16",
        "clflush size": "64",
        "cache_alignment": "64",
        "address sizes": "39 bits physical, 48 bits virtual",
        "power management": ""
      },
      {
        "processor": "4",
        "vendor_id": "GenuineIntel",
        "cpu family": "6",
        "model": "60",
        "model name": "Intel(R) Xeon(R) CPU E3-1240 v3 @ 3.40GHz",
        "stepping": "3",
        "microcode": "0x27",
        "cpu MHz": "2796.749",
        "cache size": "8192 KB",
        "physical id": "0",
        "siblings": "8",
        "core id": "0",
        "cpu cores": "4",
        "apicid": "1",
        "initial apicid": "1",
        "fpu": "yes",
        "fpu_exception": "yes",
        "cpuid level": "13",
        "wp": "yes",
        "flags": "fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt dtherm ida arat pln pts md_clear flush_l1d",
        "bugs": "cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit",
        "bogomips": "6800.16",
        "clflush size": "64",
        "cache_alignment": "64",
        "address sizes": "39 bits physical, 48 bits virtual",
        "power management": ""
      },
      {
        "processor": "5",
        "vendor_id": "GenuineIntel",
        "cpu family": "6",
        "model": "60",
        "model name": "Intel(R) Xeon(R) CPU E3-1240 v3 @ 3.40GHz",
        "stepping": "3",
        "microcode": "0x27",
        "cpu MHz": "2586.840",
        "cache size": "8192 KB",
        "physical id": "0",
        "siblings": "8",
        "core id": "1",
        "cpu cores": "4",
        "apicid": "3",
        "initial apicid": "3",
        "fpu": "yes",
        "fpu_exception": "yes",
        "cpuid level": "13",
        "wp": "yes",
        "flags": "fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt dtherm ida arat pln pts md_clear flush_l1d",
        "bugs": "cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit",
        "bogomips": "6800.16",
        "clflush size": "64",
        "cache_alignment": "64",
        "address sizes": "39 bits physical, 48 bits virtual",
        "power management": ""
      },
      {
        "processor": "6",
        "vendor_id": "GenuineIntel",
        "cpu family": "6",
        "model": "60",
        "model name": "Intel(R) Xeon(R) CPU E3-1240 v3 @ 3.40GHz",
        "stepping": "3",
        "microcode": "0x27",
        "cpu MHz": "2853.572",
        "cache size": "8192 KB",
        "physical id": "0",
        "siblings": "8",
        "core id": "2",
        "cpu cores": "4",
        "apicid": "5",
        "initial apicid": "5",
        "fpu": "yes",
        "fpu_exception": "yes",
        "cpuid level": "13",
        "wp": "yes",
        "flags": "fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt dtherm ida arat pln pts md_clear flush_l1d",
        "bugs": "cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit",
        "bogomips": "6800.16",
        "clflush size": "64",
        "cache_alignment": "64",
        "address sizes": "39 bits physical, 48 bits virtual",
        "power management": ""
      },
      {
        "processor": "7",
        "vendor_id": "GenuineIntel",
        "cpu family": "6",
        "model": "60",
        "model name": "Intel(R) Xeon(R) CPU E3-1240 v3 @ 3.40GHz",
        "stepping": "3",
        "microcode": "0x27",
        "cpu MHz": "2688.100",
        "cache size": "8192 KB",
        "physical id": "0",
        "siblings": "8",
        "core id": "3",
        "cpu cores": "4",
        "apicid": "7",
        "initial apicid": "7",
        "fpu": "yes",
        "fpu_exception": "yes",
        "cpuid level": "13",
        "wp": "yes",
        "flags": "fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt dtherm ida arat pln pts md_clear flush_l1d",
        "bugs": "cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit",
        "bogomips": "6800.16",
        "clflush size": "64",
        "cache_alignment": "64",
        "address sizes": "39 bits physical, 48 bits virtual",
        "power management": ""
      }
    ],
    "drives": [
      {
        "name": "sda",
        "fstype": null,
        "label": null,
        "uuid": null,
        "fsavail": null,
        "fsuse%": null,
        "mountpoint": null,
        "children": [
          {
            "name": "sda1",
            "fstype": "vfat",
            "label": null,
            "uuid": "4F2B-FD9B",
            "fsavail": "497,7M",
            "fsuse%": "3%",
            "mountpoint": "/boot/efi"
          },
          {
            "name": "sda2",
            "fstype": "ext4",
            "label": "root",
            "uuid": "94b9fb25-e8aa-4231-9618-35f431bcda84",
            "fsavail": "90,6G",
            "fsuse%": "53%",
            "mountpoint": "/"
          }
        ]
      },
      {
        "name": "sdb",
        "fstype": null,
        "label": null,
        "uuid": null,
        "fsavail": null,
        "fsuse%": null,
        "mountpoint": null,
        "children": [
          {
            "name": "sdb1",
            "fstype": "ext4",
            "label": "work",
            "uuid": "201b87c0-582a-4483-bc71-cb11fc6c2b90",
            "fsavail": "115,9G",
            "fsuse%": "70%",
            "mountpoint": "<redacted>"
          }
        ]
      },
      {
        "name": "sdc",
        "fstype": null,
        "label": null,
        "uuid": null,
        "fsavail": null,
        "fsuse%": null,
        "mountpoint": null,
        "children": [
          {
            "name": "sdc1",
            "fstype": "ext4",
            "label": "backup",
            "uuid": "7b11c0fd-8e69-4aa4-9758-35f1c941d1db",
            "fsavail": "644,8G",
            "fsuse%": "25%",
            "mountpoint": "<redacted>"
          }
        ]
      },
      {
        "name": "sdd",
        "fstype": null,
        "label": null,
        "uuid": null,
        "fsavail": null,
        "fsuse%": null,
        "mountpoint": null,
        "children": [
          {
            "name": "sdd1",
            "fstype": "ext4",
            "label": "games",
            "uuid": "bcfacd51-df2c-44ad-aaa8-dfd113ce2f3e",
            "fsavail": "30G",
            "fsuse%": "90%",
            "mountpoint": "/home/chris/games"
          }
        ]
      },
      {
        "name": "sde",
        "fstype": null,
        "label": null,
        "uuid": null,
        "fsavail": null,
        "fsuse%": null,
        "mountpoint": null,
        "children": [
          {
            "name": "sde1",
            "fstype": "ext4",
            "label": "storage",
            "uuid": "f76a109f-53d6-4261-a866-f27018b13fc9",
            "fsavail": "1,2T",
            "fsuse%": "60%",
            "mountpoint": "<redacted>"
          }
        ]
      }
    ],
    "ram": {
      "MemTotal": "24563760",
      "MemFree": "675520",
      "MemAvailable": "6783168",
      "Buffers": "1604916",
      "Cached": "4731608",
      "SwapCached": "0",
      "Active": "18662172",
      "Inactive": "2763100",
      "Active(anon)": "15100688",
      "Inactive(anon)": "633620",
      "Active(file)": "3561484",
      "Inactive(file)": "2129480",
      "Unevictable": "112",
      "Mlocked": "112",
      "SwapTotal": "0",
      "SwapFree": "0",
      "Dirty": "1436",
      "Writeback": "0",
      "AnonPages": "15089224",
      "Mapped": "1491116",
      "Shmem": "649080",
      "KReclaimable": "815152",
      "Slab": "1125752",
      "SReclaimable": "815152",
      "SUnreclaim": "310600",
      "KernelStack": "26348",
      "PageTables": "91160",
      "NFS_Unstable": "0",
      "Bounce": "0",
      "WritebackTmp": "0",
      "CommitLimit": "12281880",
      "Committed_AS": "27509328",
      "VmallocTotal": "34359738367",
      "VmallocUsed": "50404",
      "VmallocChunk": "0",
      "Percpu": "9600",
      "HardwareCorrupted": "0",
      "AnonHugePages": "221184",
      "ShmemHugePages": "0",
      "ShmemPmdMapped": "0",
      "CmaTotal": "0",
      "CmaFree": "0",
      "HugePages_Total": "0",
      "HugePages_Free": "0",
      "HugePages_Rsvd": "0",
      "HugePages_Surp": "0",
      "Hugepagesize": "2048",
      "Hugetlb": "0",
      "DirectMap4k": "22832688",
      "DirectMap2M": "2269184",
      "DirectMap1G": "1048576"
    },
    "dist": [
      "Ubuntu",
      "19.10",
      "Eoan Ermine"
    ],
    "arch": "x86_64",
    "kernel": "5.3.0-46-generic",
    "glxinfo": {
      "opengl_context_flags": "(none)",
      "opengl_core_profile_version": "4.5 (Core Profile) Mesa 19.2.8",
      "opengl_renderer": "Radeon RX 580 Series (POLARIS10, DRM 3.33.0, 5.3.0-46-generic, LLVM 9.0.0)",
      "name_of_display": ":0",
      "direct_rendering": "Yes",
      "opengl_vendor": "X.Org",
      "display": ":0",
      "screen": "0",
      "opengl_core_profile_shading_language_version": "4.50",
      "opengl_core_profile_context_flags": "(none)",
      "opengl_profile_mask": "compatibility profile",
      "opengl_es_profile_version": "OpenGL ES 3.2 Mesa 19.2.8",
      "opengl_core_profile_profile_mask": "core profile",
      "opengl_version": "4.5 (Compatibility Profile) Mesa 19.2.8",
      "opengl_shading_language_version": "4.50",
      "opengl_es_profile_shading_language_version": "OpenGL ES GLSL ES 3.20"
    }
  }
}
```

**Screenshots**

![grafik](https://user-images.githubusercontent.com/11466036/79325495-8de52e00-7f11-11ea-8476-ae630efad17b.png)

**Comment by stephanlachnit** [comment]
Looks like a bug from Discover to me. i18n isn't enabled in Lutris (yet).
Also see https://bugs.launchpad.net/ubuntu/+source/plasma-discover/+bug/1769149,

**Comment by stephanlachnit** [comment]
Actually it might be related to #2750.

**Comment by DaDummy** [comment]
FYI: I'm not sure as to how to test this with a downloaded package, but I will report back once a fix for #2750 has landed in the repositories or when a similar issue arises with other packages, as that would greatly decrease the likelihood of this issue being rooted in lutris packaging itself.

Though just to be clear: This appears to be a pure display bug limited to (certain) package manager GUIs.

**Comment by DaDummy** [comment]
The fix for #2750 (#2751) appears to have solved the issue. Thanks :)
